### DISCUSSION TITLE
Whitelist websites for SO updates

### DISCUSSION THREAD

**Discussion by martin8615** [Security-Onion-Solutions/securityonion#2042] [Unsupported Versions] [answered]
We are using a proxy to connect our grit to the internet and are trying to update to 2.3.10.
The last time we updates our system half of the installation went fine but the other half couldn´t be downloaded. So we´ve got an broken installation and reinstalled our grit.

Which sites or IPs do I have to whitelist in our proxy/firewall that running `soup `or` so-rule-update` works properly?

**Comment by dougburks** [comment, marked as answer]
@weslambert created issue https://github.com/Security-Onion-Solutions/securityonion/issues/2046 to improve our documentation.
